frontend/package.json

```diff
@@ -174,54 +176,30 @@
       "plugin:jsx-a11y/recommended"
     ]
   },
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,ts,tsx}",
-      "!<rootDir>/node_modules/",
-      "!<rootDir>/build/",
-      "!src/**/*.stories*.{js,jsx,ts,tsx}"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "statements": 43.34,
-        "branches": 37.61,
-        "functions": 38.09,
-        "lines": 43.16
-      }
-    },
-    "coverageReporters": [
-      "text"
-    ],
-    "transformIgnorePatterns": [
-      "/node_modules/(?!d3|internmap|react-markdown|xterm|github-markdown-css|vfile|unist-.+|unified|bail|is-plain-obj|trough|remark-.+|mdast-util-.+|micromark|parse-entities|character-entities|property-information|comma-separated-tokens|hast-util-whitespace|remark-.+|space-separated-tokens|decode-named-character-reference|spacetime|monaco-editor.+|react-monaco-editor|@monaco-editor/react)"
-    ],
-    "resetMocks": false
-  },
   "prettier": "@kinvolk/eslint-config/prettier-config",
   "devDependencies": {
     "@axe-core/react": "^4.3.2",
     "@storybook/addon-actions": "7.6.7",
     "@storybook/addon-essentials": "7.6.7",
     "@storybook/addon-links": "7.6.7",
-    "@storybook/addon-storyshots": "7.6.7",
+    "@storybook/builder-vite": "^8.1.0",
     "@storybook/node-logger": "7.6.7",
-    "@storybook/preset-create-react-app": "^7.6.7",
     "@storybook/react": "7.6.7",
     "@storybook/react-webpack5": "^7.6.7",
+    "@testing-library/jest-dom": "^6.4.5",
     "@testing-library/user-event": "^14.5.1",
     "@types/nock": "^11.1.0",
     "@types/redux-mock-store": "^1.0.4",
+    "@vitest/coverage-v8": "^1.6.0",
     "fetch-mock": "^9.11.0",
     "http-proxy-middleware": "^2.0.1",
     "husky": "^4.3.8",
-    "i18next-parser": "^4.7.0",
-    "jest-canvas-mock": "^2.4.0",
-    "jest-html-reporter": "^3.10.2",
-    "jest-websocket-mock": "^2.5.0",
+    "i18next-parser": "^8.13.0",
```

Comment:
had a bunch of CommonJS import errors with the old i18next-parser version so I updated it 
I don't see a lot of breaking changes, just some improvements in the way plural is handled, you can see updates to the translation files